CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
-# Version 1.8.00
+# Version 1.7.10
 
 ### Bug fixes
 - MASFoundation's OTP channel selection was returning an internal error to the original request. The Mobile SDK now sends only cancellation of OTP process, and/or results of the original request. [DE366491]
+- The Mobile SDK was causing an error during device registration, "The device has already been registered." which prevents users to re-register the device using different credentials when share keychain was incorrectly configured. The Mobile SDK improves the device registration process to minimize this type of error. [DE369778]
+- The Mobile SDK reserved "Content-Type" and "Accept" header for requests and did not provide an option for developers to override these values. The Mobile SDK now allows developers to override the values as needed. [DE369138]
+- The Mobile SDK caused an incorrect error for session lock using multiple applications using SSO. The Mobile SDK now handles session lock's status properly, and no longer causes the issue. [DE374443]
```

Comment:
The Mobile SDK displayed an incorrect error for session lock when multiple applications are using SSO. The Mobile SDK now provides accurate session lock status.